src/bepasty/tests/test_app.py

```diff
@@ -24,3 +42,61 @@ def test_secret_key():
     }
     app = create_app()
     assert app.config['SECRET_KEY'] != secret_key
+
+
+class TestView(MethodView):
+    callback = None
+
+    def get(self):
+        TestView.callback()
+        return 'done'
+
+
+def prepare(callback):
+    app = create_app()
+    app.add_url_rule('/test_call', view_func=TestView.as_view('test.test_call'))
+
+    TestView.callback = staticmethod(callback)
+    client = app.test_client()
+
+    assert app.config['APP_BASE_PATH'] == Config.APP_BASE_PATH
+
+    return app, client
+
+
+def test_none(app_fixture):
+    Config.APP_BASE_PATH = None
+
+    def none_callback():
+        url = url_for('test.test_call')
+        assert url == request.path
+        url = url_for('test.test_call', _external=True)
+        assert url == 'http://localhost' + request.path
```

Comment:
hmm, isn't that getting into the "testing flask (not: bepasty)" department?

Reply:
this was from proxy code test. removed _external test